src/components/templates/Auditorium/components/Section/constants.ts

```diff
@@ -0,0 +1,9 @@
+export const DEFAULT_ROWS_NUMBER = 17;
+export const DEFAULT_COLUMNS_NUMBER = 23;
+
+export const VIDEO_MIN_WIDTH_IN_SEATS = 17;
+
+// If you change this, make sure to also change it in Section.scss
+export const SEAT_SIZE = "var(--seat-size)";
+export const SEAT_SIZE_MIN = "var(--seat-size-min)";
+export const SEAT_SPACING = "var(--seat-spacing)";
```

Comment:
renamed them in https://github.com/sparkletown/sparkle/pull/1251/commits/c7e60d97eac7ff85f121f5b5d8a480f8f16c3d70

Reply:
When Auditorium is out, Audience will be considered a legacy

Reply:
> When Auditorium is out, Audience will be considered a legacy

Have you ensured that all changes/fixes/etc that have since happened to `Audience` have been copied here as well? And can you guarantee they are going to stay in sync?

This is why I really dislike copy/pasted code, it's way too easy for it to fall out of sync and just cause bigger duplicate headaches/recurring bugs/etc.

Reply:
```
Have you ensured that all changes/fixes/etc that have since happened to Audience have been copied here as well? 
```
I don't remember any changes/fixes to Audience happening since I created this PR.

```
And can you guarantee they are going to stay in sync?
```
Since we have thorough code reviews, it should be the case. + We should fairly quickly get rid of using Audience

Reply:
_nods_ fair enough.

Is there any reason we wouldn't be able to get rid of Audience as part of this PR out of curiosity? I guess that might increase the risk/etc of a 'breaking issue'.

Reply:
`Is there any reason we wouldn't be able to get rid of Audience` they have slightly different API, I would approach this in a less radical way. I will tag Audience as deprecated in the codebase + will add the comment in the Admin panel venue creation that you should you Auditorium instead

Reply:
@0xdevalias I can't deprecate Audience until we have a good UI to control Auditorium creation/addition. I laid out the plan here: https://github.com/sparkletown/internal-sparkle-issues/issues/636

Resolving this for now, feel free to open

Reply:
Plan sounds good to me.